lib/active_shipping/carriers/usps.rb

```diff
@@ -168,15 +168,15 @@ class USPS < Carrier
       /Delivery status information is not available/
     ]
 
-    ESCAPING_AND_SYMBOLS = /&amp;lt;\S*&amp;gt;/
-    LEADING_USPS = /^USPS/
+    ESCAPING_AND_SYMBOLS = /&lt;\S*&gt;/
+    LEADING_USPS = /^USPS /
     TRAILING_ASTERISKS = /\*+$/
     SERVICE_NAME_SUBSTITUTIONS = /#{ESCAPING_AND_SYMBOLS}|#{LEADING_USPS}|#{TRAILING_ASTERISKS}/
 
     def find_tracking_info(tracking_number, options = {})
       options = @options.update(options)
       tracking_request = build_tracking_request(tracking_number, options)
-      response = commit(:track, tracking_request, (options[:test] || false))
+      response = commit(:track, tracking_request, options[:test] || false)
```

Comment:
Anyway, that's for a different PR. I changed it to be simply `options[:test]`. Boolean coercion is not necessary here. It's use din `commit`:
``` ruby
    def commit(action, request, test = false)
      ssl_post("#{test ? TEST_URL : LIVE_URL}/#{RESOURCES[action]}", request)
    end
```